tests/e2e/cucumber/environment/index.ts

```diff
@@ -173,3 +175,9 @@ const cleanUpGroup = async (adminUser: User) => {
   await Promise.all(requests)
   createdGroupStore.clear()
 }
+
+const removeTempUploadDirectory = () => {
+  if (fs.existsSync(config.tempAssetsPath)) {
+    fs.rmSync(config.tempAssetsPath, { recursive: true })
```

Comment:
will it remove the temp directory or the content inside the temp directory?

Reply:
removes the directory as well